cibuild.sh

```diff
@@ -14,7 +20,36 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-cd src/Compilers/CSharp/csc
+echo Compiling using toolset compiler 
+docompile
+if [ $? -ne 0 ]; then
```

Comment:
Open to suggestions.  Been using bash because I know that it will work and be present on the Linux machine.  

Note: This entire script is meant to be a temporary solution.  Once we close the functionality gap between Linux and Windows we'll be moving as much as possible into a shared infrastructure.  Hopefully the script will just be one or two lines to fire up the shared validation code.